web/src/left_sidebar_navigation_area.js

```diff
@@ -117,3 +119,14 @@ function do_new_messages_animation($li) {
 export function initialize() {
     update_scheduled_messages_row();
 }
+
+export function narrow_to_inbox() {
+    remove($(".top_left_all_messages"));
+    remove($(".top_left_starred_messages"));
+    remove($(".top_left_recent_topics"));
+    remove($(".top_left_mentions"));
+    $(".top_left_inbox").addClass("active-filter");
+    setTimeout(() => {
+        resize.resize_stream_filters_container();
+    }, 0);
```

Comment:
Do you understand why the `setTimeout` is needed here? I see it in `narrow_to_recent_topics` as well, but no comment.

As a side note, I think `narrow` is a term that is intended to only apply to a message feed view; it comes from "narrowing the set of messages you're looking at from the All messages". We're probably going to replace it with something more about "views" in user-facing text, but I think we shouldn't be using it as a term at all when talking about the inbox view; I think something like `left_sidebar_navigation_area.highlight_inbox_view` would be more appropriate, and similarly for the recent conversations one -- feel free to do a little prep PR cleaning up this aspect of naming in this module.

Reply:
Done.